examples/guestbook/guestbook_ns/frontend-controller.json

```diff
@@ -0,0 +1,46 @@
+{
+    "kind": "ReplicationController",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "frontend-controller",
+        "labels": {
+            "name": "frontend",
+            "template": "frontend"
```

Comment:
"template" isn't the right idea here. Besides, do we even need it? I'd maybe remove it. Alternatively, we could apply some of the typically labels we've discussed, such as those mentioned here: https://github.com/GoogleCloudPlatform/kubernetes/pull/3233#discussion_r22664461